docs/content/preview/drivers-orms/go/yb-pgx.md

```diff
@@ -143,6 +146,8 @@ You can specify multiple hosts in the connection string to provide alternative o
 postgres://username:password@host1:5433,host2:5433,host3:5433/database_name?load_balance=true
 ```
 
+To query the cluster for a list of available hosts, use the `yb_servers()` function.
```

Comment:
We don't expect users to query the yb_servers() function. It is the driver who does it. This statement may confuse readers. Same holds true on other language pages.